devops/all-tests-job-template.yml

```diff
@@ -29,12 +38,9 @@ jobs:
 
   - template: python-infra-upgrade-steps-template.yml
 
-  - script: python scripts/requirements_pin.py --input requirements.txt --output requirements-fixed.txt --loglevel INFO
-    displayName: "Pin Requirements file"
-    condition: eq('${{parameters.pinRequirements}}', True)
+  - template: requirements-pin-steps-template.yml
 
-  - script: pip install -r ${{ parameters.requirementsFile }}
-    displayName: 'Install required packages specified in ${{ parameters.requirementsFile }}'
+  - temaplte: pip-install-steps-template.yml
```

Comment:
`template` ?

Reply:
Wow, well spotted!

Reply:
Not that I've ever made that particular typo myself, you understand......